src/clarity-angular/layout/vertical-nav/vertical-nav-group.html

```diff
@@ -1,18 +1,19 @@
-<button
-    class="nav-group-trigger"
-    [class.active]="active && !expanded"
-    type="button"
-    (click)="toggleExpand()">
+<div class="nav-group-content" (click)="onNavContentClick()">
     <ng-content select="[clrVerticalNavIcon]"></ng-content>
-    <span class="nav-text">
+    <ng-content select="[clrVerticalNavLink]"></ng-content>
+    <div class="nav-group-text">
         <ng-content></ng-content>
-    </span>
-    <clr-icon class="clr-nav-caret"
-              shape="caret"
-              [attr.dir]="(this.expanded) ? 'down' : 'right'"></clr-icon>
-</button>
+    </div>
+    <button
+        class="nav-group-trigger"
+        type="button"
+        (click)="toggleExpand()">
+        <clr-icon shape="caret"
+                  [attr.dir]="(this.expanded) ? 'down' : 'right'"></clr-icon>
+    </button>
+</div>
 <div class="nav-group-children"
      [@clrExpand]="expandAnimationState"
      (@clrExpand.done)="expandAnimationDone($event)">
```

Comment:
Can you please add a comment saying that when `clrIfExpanded` handles animations properly, we should remove this extra wrapper and just add the class `.nav-group-children` to the `<clr-vertical-nav-group-children>` host instead? Thanks.